packages/ts/react-crud/src/autoform.tsx

```diff
@@ -286,6 +286,23 @@ export function AutoForm<M extends AbstractModel>({
     }
   }, [item]);
 
+  function handleSubmitError(error: unknown) {
+    if (error instanceof ValidationError) {
+      const nonPropertyError = error.errors.find((validationError) => !validationError.property);
```

Comment:
What happens if more than one record level validation fails? We only show one of the errors sent back from the endpoint?

Reply:
Will show all errors

Reply:
Would it make sense that instead of joining error messages with `'; '` we render them under the form like this:
```
There are validation errors in the form:
 * foobar
 * just a message
 * ...
``` 
I think this improves the UX to make the errors more understandable. WDYT?

Reply:
Added this options